src/doc/src/reference/registry-index.md

```diff
@@ -140,26 +140,24 @@ explaining the format of the entry.
             "req": "^0.6",
             // Array of features (as strings) enabled for this dependency.
             "features": ["i128_support"],
-            // Boolean of whether or not this is an optional dependency.
+            // Boolean of whether or not this is an optional dependency. Defaults to `false` if not specified.
```

Comment:
Can this be more precise? Something like:
```
If not specified:
- cargo >= 1.8x: defaults this as `false`.
- cargo >= 1.19: ignore the one version.
- cargo <= 1.18: ignore the entire package.
```
Or
```
Current versions of cargo defaults to `false` if not specified.
Cargos before 1.8x the version will be ignored.
Very old cargos before 1.19 ignore the entire package.
```
When someone looks to skip this data on crates.io, this is where I would look to see if it is ok. So I want to leave a note that it is not ok for the git index.